docs/content/DevelopmentPlan/DevelopmentPlan.md

```diff
@@ -7,141 +7,333 @@ date: "2024-09-16"
+
+Editor of choice should depends on whatever is the one that makes you move the fastest. For me, it is [Neovim](https://neovim.io/)
+
+### documentation
+
+Generated with [Quartz](https://quartz.jzhao.xyz/), hosted on GitHub Pages.
+
+One can use any file-based markdown editor to update documentation, but recommend to use [Obsidian](https://obsidian.md/)
+
+Design file will be using [Figma](https://figma.com) for high-fidelity prototype, [Excalidraw](https://excalidraw.com/)
+for low-fidelity prototype.
+
+Citations will use [Chicago Manual of Style](https://www.chicagomanualofstyle.org/tools_citationguide.html) with
+a custom [CSL](https://github.com/aarnphm/tinymorph/blob/main/docs/content/chicago-fullnote-bibliography.csl) for supporting footnotes
+
+### web-based editor (morph)
+
+Language: JS/TS
+
+Stack:
+
+- rendering: Any React-based framework, e.g. Next, w/ Node backend.
+- styling: SASS, framer-motion, @shadcn/ui
+- testing: Jest
 
-The implementation decisions can, and likely will, change over the course of the project. The initial documentation should be written in an abstract way; it should be agnostic of the implementation choices, unless the implementation choices are project constraints. However, recording our initial thoughts on implementation helps understand the challenge level and feasibility of a project. It may also help with early identification of areas where project members will need to augment their training.
+Formatter: prettier
 
-Topics to discuss include the following:
+Linter: eslint
 
-- Specific programming language
-- Specific libraries
-- Pre-trained models
-- Specific linter tool (if appropriate)
-- Specific unit testing framework
-- Investigation of code coverage measuring tools
-- Specific plans for Continuous Integration (CI), or an explanation that CI is not being done
-- Specific performance measuring tools (like Valgrind), if appropriate
-- Tools you will likely be using?
+Bundler/Transpiler: vite/esbuild
 
-git, GitHub and GitHub projects should be part of your technology.
+CI: include styling and linter, possibly a small set of A/B tests.
+
+tool: [`pnpm`](https://pnpm.io/) for monorepo workspace management
+
+### ml inference server (asteraceae)
+
+Language: Python
+
+Stack:
+
+- online inference: BentoML, vLLM, pydantic
+- offline inference: [llama.cpp](https://github.com/ggerganov/llama.cpp)
+- testing: pytest
+
+Linter: ruff
+
+CI: include styling and linter, regression testing, e2e testing.
+
+Model: Gemma 2 for SAE exploration, Llama 3.1 for final models (for POC specifically, will also train a few LoRAs for writers tonality)
+
+Tool: [`uv`](https://github.com/astral-sh/uv) for monorepo management
 
 ## Coding Standard
 
-What coding standard will you adopt?
+### TS/JS
+
+[Google style guide](https://google.github.io/styleguide/) will be used.
+
+All related tools has been setup corespondingly.
+
+### Python
+
+a variation of Google style guide will be used.
```

Comment:
Capitalize the first letter: "A variation of the Google style guide will be used."

Reply:
updated